controller/workitem.go

```diff
@@ -241,7 +241,9 @@ func (c *WorkitemController) Reorder(ctx *app.ReorderWorkitemContext) error {
 			hasChildren := workItemIncludeHasChildren(appl, ctx)
 			wi2 := ConvertWorkItem(ctx.RequestData, *wi, hasChildren)
 			dataArray = append(dataArray, wi2)
+			log.Info(ctx, nil, "Reordered item: %v", wi2)
```

Comment:
@xcoulon I guess you don't want to log all the work item value. Right ? If you want to log something try using its ID, space_id, etc..